src/app/shared/hero/hero.component.css

```diff
@@ -36,7 +36,7 @@
 }
 .toolbar section {
   display: flex;
-  align-items: center;
```

Comment:
That extra line of text on the Story page also messes up the toolbar-affixing a bit.  (I think that affix stuff is currently based on a fixed-height toolbar).